src/lib/components/Swap/Toolbar/index.tsx

```diff
@@ -18,7 +18,7 @@ const ToolbarRow = styled(Row)`
 `
 
 export default memo(function Toolbar({ disabled }: { disabled?: boolean }) {
```

Comment:
I think you can drop disabled entirely w/ this logic. I don't recall off the top of my head what this disabled state is usually caused by, so you might want to double check and make sure we don't *ever* want a `disabled`-contingent UI state from the toolbar.

Reply:
It's caused by an `!useActiveWeb3React().active`, but that's already been baked into the Toolbar. Removed!